third_party/move/move-compiler-v2/transactional-tests/tests/no-v1-comparison/dotdot/nested.exp

```diff
@@ -0,0 +1,58 @@
+processed 5 tasks
+
+task 0 'publish'. lines 1-51:
+Error: compilation errors:
+ error: match not exhaustive
+   ┌─ TEMPFILE:22:10
+   │
+22 │         match (x) {
```

Comment:
Why that? Should not happen if I looke at around line 22

Reply:
This is tracked in https://github.com/aptos-labs/aptos-core/issues/14296